schema_based_on_diagram.sql

```diff
@@ -0,0 +1,42 @@
+-- Clinic Database Schema
+CREATE TABLE medical_historeis (
+  id BIGSERIAL NOT NULL PRIMARY KEY,
+  admitted_at TIMESTAMP,
+  patient_id INT NULL REFERENCES patients(id) ON DELETE CASCADE,
+  status VARCHAR(40) NOT NULL
+);
+
+CREATE TABLE invoices (
+  id BIGSERIAL NOT NULL PRIMARY KEY,
+  total DECIMAL NOT NULL,
+  generated_at TIMESTAMP,
+  payed_at TIMESTAMP,
+  medical_history_id INT NULL REFERENCES medical_historeis(id) ON DELETE CASCADE
+);
+
+CREATE TABLE patients (
+  id BIGSERIAL NOT NULL PRIMARY KEY,
+  name VARCHAR(100) NOT NULL,
+  date_of_birth DATE NOT NULL 
+);
+
+CREATE TABLE invoice_items (
+  id BIGSERIAL NOT NULL PRIMARY KEY,
+  unit_price DECIMAL NOT NULL,
+  quantity BIGSERIAL NOT NULL,
+  total_price DECIMAL NOT NULL,
+  invoices_id INT NULL REFERENCES invoices(id) ON DELETE CASCADE,
+  treatment_id INT NULL REFERENCES treatment(id) ON DELETE CASCADE
+)
+
+CREATE TABLE tretments (
+  id BIGSERIAL NOT NULL PRIMARY KEY, 
+  type VARCHAR(40) NOT NULL,
+  name VARCHAR(40) NOT NULL
+)
+
```

Comment:
- [ ] nice work. please ensure all "join table" for the many-to-many relastionships are created. eg table between `medical_histories` and `treatments`

Reply:
@tufoinnkuo10 Thanks for the reminder.